AOB-1455: Upgrade onboarder installation and troubleshooting doc (#1754)
akeneo/pim-docs: onboarder/troubleshooting/index.rst

    :scale: 100 %
    :alt: Result of the check pubsub access command
    :align: center

> Proposals from Onboarder don't show up into the PIM

Reviewer: ```suggestion
Proposals from Onboarder don't show up in the PIM
```

---------------------------------------------------

> If you don't received proposals into the PIM, double check that you have rightfully setup and configure your workers, see :doc:`synchronization </onboarder/synchronization/index>` page.

Reviewer: ```suggestion
If you don't receive proposals in the PIM, double check that you properly set up and configured your workers, see :doc:`synchronization </onboarder/synchronization/index>` page.
```
"rightfully" means "à juste titre"^^

Author: Heureusement que t'es là pour corriger mon anglais :D
